app/frontend/src/components/views/workspace-form/WorkSpaceForm.tsx

```diff
@@ -0,0 +1,100 @@
+import styled from 'styled-components';
```

Comment:
pls, rename the `WorkSpaceForm` to the `WorkspaceForm`

Reply:
Fixed